src/App.js

```diff
@@ -34,6 +34,19 @@ class App extends React.Component {
       base64DynamoBackground = backgroundImage;
   }
 
+  componentDidMount() {
+    document.addEventListener('keydown', this.handleKeyDown);
+    //TODO : As alternative we can receive the event from the Childs like the Static component
```

Comment:
Currently we are listening to the keyboard in both places, the Static component and the App component, for the first one we detect only the Enter key to call the importSettings, for the second one we detect only the Esc key to call the closeDynamo. Instead of listening to the keyboard of the App component, We can listen only from the static (child) for both cases, the Enter and Esc, and when we will detect the Esc is pressed it will raise an event to its parent component (App) so it can deal with the close Dynamo there. I see this approach more cleaner as part of a future refactoring.